Adds LTI dashboard to monorepo

stopfstedt opened the pull request:
please also check the last couple of commits

```
stefan@nichtsnutz: ~/dev/projects/frontend on add_lti-dashboard_to_monorepo
$ gitpretty
* 41c6fc769 - (HEAD -> add_lti-dashboard_to_monorepo, origin/add_lti-dashboard_to_monorepo) c/p CoC and License file from ilios-common. (6 minutes ago) <Stefan Topfstedt>
* 5c0a87584 - adds README file back to lti-dashboard package. (8 minutes ago) <Stefan Topfstedt>
* b73adf94a - adds various lti-dashboard specific workflows. (21 minutes ago) <Stefan Topfstedt>
* 9ff96e1af - actually ignore. (25 minutes ago) <Stefan Topfstedt>
*   f8b1d9ec4 - Merge remote-tracking branch 'lti-dashboard/master' into add_lti-dashboard_to_monorepo (38 minutes ago) <Stefan Topfstedt>

```

jrjohnson commented on .github/dependabot.yml:
```diff
@@ -17,35 +17,35 @@ updates:
       patterns:
       - "@embroider/*"
   ignore:
-  - dependency-name: "@babel/core"
```
Not sure about this `yaml` spacing change. I think lists are supposed to be unindented.

stopfstedt replied:
corrected :white_check_mark:

jrjohnson reviewed (changes requested):
`lti-dashboard-tag-version.yaml` and `lti-dashboard-create_release.yaml` should be removed they'll do bad things to our existing frontend identical processes. 

The `README` is good, though we probably want to consolidate all the setup and get going steps into the main `README` the COC and LICENSE can go away though, we don't publish this to npm, and I don't think our build has ever shipped this info off to AWS.

stopfstedt commented:
@jrjohnson - i removed the requested files and fixed the indentation issue. 

then merged `master` in and resolved a merge-conflict with the pnpm lock file that arose in the mean time.

this is ready for re-review.
